radixdlt/src/test/java/com/radixdlt/consensus/liveness/PacemakerImplTest.java

```diff
@@ -185,41 +187,52 @@ public void when_inserting_valid_but_old_new_views__then_no_new_view_is_returned
 		ScheduledExecutorService executorService = getMockedExecutorService();
 		PacemakerImpl pacemaker = new PacemakerImpl(TEST_PACEMAKER_TIMEOUT, executorService);
 		pacemaker.processQC(View.of(0));
-		assertThat(pacemaker.processNewView(newView, validatorSet)).isEmpty();
+		assertThat(pacemaker.processNewView(newView, validatorSet, mock(ProposerElection.class))).isEmpty();
 	}
 
 	@Test
 	public void when_inserting_current_and_accepted_new_views__then_qc_is_formed_and_current_view_has_changed_and_no_new_timeout() {
-		View view = View.of(0);
+		View view = View.of(1);
 		NewView newView = makeNewViewFor(view);
 		ValidatorSet validatorSet = mock(ValidatorSet.class);
 		ValidationResult result = mock(ValidationResult.class);
 		when(result.valid()).thenReturn(true);
 		when(validatorSet.validate(any(), any())).thenReturn(result);
 		ScheduledExecutorService executorService = getMockedExecutorService();
 		PacemakerImpl pacemaker = new PacemakerImpl(TEST_PACEMAKER_TIMEOUT, executorService);
-		assertThat(pacemaker.processNewView(newView, validatorSet)).isPresent().get().isEqualTo(View.of(0));
-		assertThat(pacemaker.getCurrentView()).isEqualTo(View.of(0));
-		verify(executorService, times(0)).schedule(any(Runnable.class), anyLong(), any());
+		pacemaker.processQC(View.of(0));
+
+		assertThat(pacemaker.processNewView(newView, validatorSet, mock(ProposerElection.class))).isPresent().get().isEqualTo(View.of(1));
+		assertThat(pacemaker.getCurrentView()).isEqualTo(View.of(1));
```

Comment:
The method is called `when_inserting_current_and_accepted_new_views__then_qc_is_formed_and_current_view_has_changed_and_no_new_timeout` but it doesn't actually test for new timeouts